bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+'''
+Usage -
+The input file should be a JSON file in the format {"key": 100, "key2": 200 ...}. 
+The script will generate two output JSON files (Evergreen and Atlas compatible) based on this input data.
+'''
+
+import argparse
+import json
+
+
+def parse_input_file(input_file):
+    ''' Read and parse the input JSON file '''
+    with open(input_file, 'r') as f:
+        input_data = json.load(f)
+    return input_data
+
+
+def generate_output_atlas(test_name, input_data):
+    ''' Generate the atlas-compatible JSON file '''
+
+    output_data = {
+        "Test Name": test_name,
+        "metrics": [
+            {"name": key, "value": value}
+            for key, value in input_data.items()
+        ],
+        "config": {}
+    }
+    return output_data
+
+
```

Comment:
Two new lines after every function seem excessive. We usually only put one.

Reply:
This is a python convention. We follow it in Wiredtiger.